sponsorship_switzerland/models/completion_rules.py

```diff
@@ -131,6 +136,32 @@ def get_from_partner_ref(self, stmts_vals, st_line):
                 )
         return res
 
+    def recurring_contract_wire_transfer_mapping(self, stmts_val, st_line):
+        if "name" not in st_line:
+            return
+
+        name = st_line['name']
+        res = dict()
+
+        wire_transfer_mode = self.env['account.payment.mode'].search([
```

Comment:
The name is translated, we want to find it even if the user importing is not using English.
```suggestion
        wire_transfer_mode = self.env['account.payment.mode'].with_context(lang="en_US").search([
```